drivers/lcd/ili9341.c

```diff
@@ -287,20 +287,26 @@
 
 /* General fix display resolution */
 
-#define ILI9341_XRES           240
-#define ILI9341_YRES           320
+#if CONFIG_LCD_ILI9341_NINTERFACES < 1
```

Comment:
I think this is not necessary because the menuconfig already defines LCD_ILI9341_NINTERFACES to range 1 - 2. So LCD_ILI9341_NINTERFACES < 1 is an error

Reply:
Sure,  I always trying to make more defensive ;)